charts/kube-prometheus-stack/templates/prometheus-operator/deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
     metadata:
       labels:
         app: {{ template "kube-prometheus-stack.name" . }}-operator
+        app.kubernetes.io/name: {{ template "kube-prometheus-stack.name" . }}-prometheus-operator
+        app.kubernetes.io/component: prometheus-operator
```

Comment:
Any particular reason for not using `{{ include "kube-prometheus-stack.labels" . | indent x }}` as we do everywhere else?

Reply:
You are right, the label are used everywhere, including on dashboard which has nothing todo with prometheus-operator.